files/multifilereader_binary_go119_test.go

```diff
@@ -0,0 +1,10 @@
+//go:build !go1.20
```

Comment:
:thought_balloon:  @hacdias would it be better to have `go1.19` here to avoid the problem when we try to build with `go1.21`? my understanding is that 21 will behave the same way as 20, and we don't need to worry about <19.
```suggestion
//go:build go1.19
```

Reply:
the `go1.20` tag will be set in 1.21 too.
Your suggestion is not wrong it just do nothing.

Reply:
I'm marking this as resolved due to @Jorropo explanation.